tutorials/realm/Barrier.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: page
+permalink: /tutorial/realm/barrier.html
```

Comment:
This is pretty minor, but I just realize the name of this file is capitalized (`Barrier.md`) but the file it generates is lowercase (`barrier.html`). For consistency, can we just keep everything lowercase?

That is, rename `Barrier.md` to `barrier.md` (and so on throughout the tutorial).

Reply:
fixed